src/ocean/time/Time.d

```diff
@@ -166,7 +166,7 @@ struct TimeSpan
          * Params: t = A TimeSpan value to add
          * Returns: a copy of this instance after adding t.
          */
-        TimeSpan opAddAssign(TimeSpan t)
+        TimeSpan opOpAssign (string op) (TimeSpan t) if (op == "+")
```

Comment:
The `+` and `-` operators look very similar, why no use common string mixin trick?
```d
    TimeSpan opBinary (string op) (TimeSpan t)
        if (op == "+" || op == "-")
    {
        mixin("return TimeSpan(ticks_ " ~ op ~ " t.ticks_);");
    }
```

Reply:
Worth noting that the main reason why the D1 overloads were replaced with the D2 syntax, is so that this trick can be used. It was noticed that practically every class which overloaded operators had a huge amount of code duplication.